presto-main/src/test/java/com/facebook/presto/execution/scheduler/TestSourcePartitionedScheduler.java

```diff
@@ -332,8 +347,9 @@ public void testBalancedSplitAssignment()
         StageScheduler firstScheduler = getSourcePartitionedScheduler(firstPlan, firstStage, nodeManager, nodeTaskMap, 200);
 
         ScheduleResult scheduleResult = firstScheduler.schedule();
-        assertTrue(scheduleResult.isFinished());
+        assertEffectivelyFinished(scheduleResult, firstScheduler);
         assertTrue(scheduleResult.getBlocked().isDone());
+        assertEquals(scheduleResult.getBlockedReason(), Optional.of(NO_ACTIVE_DRIVER_GROUP));
```

Comment:
This is unnecessary now that you are using assertEffectivelyFinished: `assertTrue(scheduleResult.getBlocked().isDone())`.

This is testing the implementation details instead of the behaviors. It doesn't matter what the blocked reason is when it's not blocked. `assertEquals(scheduleResult.getBlockedReason(), Optional.of(NO_ACTIVE_DRIVER_GROUP));`

Reply:
Correct. I thought I removed all of these assertion but left this one